source/linux/Foundational_Components/Tools/Flash_via_UART.rst

```diff
@@ -2,124 +2,20 @@
 Flash via UART
 ##############
 
-UART is used as the transport or interface to send the file to flash to the EVM.
+This tool uses UART as the transport or interface to send the file to flash to the EVM.
 
-***************
-Important files
-***************
+The UART Flash tool is part of MCU+SDK installer and can be downloaded from |__SDK_DOWNLOAD_URL__|.
 
-:file:`uart_uniflash.py` is located at :file:`<TI_SDK_PATH>/bin/uart_uniflash/uart_uniflash.py`
-
-Flashing application and example flash configuration files for GP, HS-FS and HS
-are located at
-
-.. ifconfig:: CONFIG_part_variant in ('AM64X')
-
-    :file:`<TI_SDK_PATH>/bin/uart_uniflash/bin/am64xx-evm/`
+Detailed instructions on using the tool can be found in MCU+SDK docs.
 
 .. ifconfig:: CONFIG_part_variant in ('AM62X')
 
-    :file:`<TI_SDK_PATH>/bin/uart_uniflash/bin/am62xx-evm/`,
-    :file:`<TI_SDK_PATH>/bin/uart_uniflash/bin/am62xx-lp-evm/`,
-    :file:`<TI_SDK_PATH>/bin/uart_uniflash/bin/am62xxsip-evm/`
+    `MCU+SDK Flash Tools Documentation  <https://software-dl.ti.com/mcu-plus-sdk/esd/AM62X/latest/exports/docs/api_guide_am62x/TOOLS_FLASH.html>`__.
 
-.. ifconfig:: CONFIG_part_variant in ('AM62AX')
+.. ifconfig:: CONFIG_part_variant in ('AM64X')
```

Comment:
Has AM62A been missed here?

Reply:
@AashvijShenai AM62A SDK is not part of our charter. We do not package any tools in AM62A SDK as of now.